client/src/modules/cost_center/allocation_bases/allocation_bases.js

```diff
@@ -0,0 +1,107 @@
+angular.module('bhima.controllers')
+  .controller('CostCenterAllocationBasesController', CostCenterAllocationBasesController);
+
+CostCenterAllocationBasesController.$inject = [
+  'CostCenterService',
+  'uiGridConstants',
+  'NotifyService',
+];
+
+function CostCenterAllocationBasesController(
+  CostCenters,
+  uiGridConstants,
+  Notify,
+) {
+  const vm = this;
+
+  // global variables
+  vm.gridApi = {};
+  vm.filterEnabled = false;
```

Comment:
I don't think you use this line.  Could you either put an online filter on the grid, or remove it?